.github/workflows/composed-gradle-nexus-iq.yaml

```diff
@@ -90,4 +90,4 @@ jobs:
           echo "APPLICATION_ID=${{ vars.ORGANIZATION }}-${REPO_DASHED}" >> "$GITHUB_ENV"
       - name: Sonatype Lifecycle Scan
         # yamllint disable-line rule:line-length
-        uses: lfit/releng-reusable-workflows/.github/workflows/reuse-sonatype-lifecycle.yaml@484ea7e8c8c880ca1905ee837dcf6d6547ea0418
+        uses: lfit/releng-reusable-workflows/.github/workflows/reuse-sonatype-lifecycle.yaml@05bc79da997ed6406fb58d43508fd01ff27a2bb8 # 2024-12-10
```

Comment:
What is this date here for? should it not be version no?

Reply:
Until we tag a release, we're pinning to commits without corresponding tagged releases. Seems a bit odd to not be commenting at all when we're doing that, but I've removed this for now.